package.json

```diff
@@ -69,10 +69,15 @@
     "yargs": "^17.7.2"
   },
   "scripts": {
+    "install:rust": "curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y",
+    "install:rust:windows": "winget install Microsoft.VisualStudio.2022.Community --silent --override \"--wait --quiet --add ProductLang En-us --add Microsoft.VisualStudio.Workload.NativeDesktop --includeRecommended\" && winget install Rustlang.Rustup",
+    "install:wasm-pack:sh": ". $HOME/.cargo/env && curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh -s -- -y",
+    "install:wasm-pack": "cargo install wasm-pack",
```

Comment:
I would use `cargo binstall wasm-pack` which downloads a binary (binstall = binary install). You have to first make sure that cargo binstall itself is installed, via `cargo install cargo-binstall`.

Reply:
@adamchalmers does that also work for platforms that might not have the binary precompiled somewhere and have to compile? Thinking about Windows on ARM and potentially Linux

Reply:
@adamchalmers Looking into binstall got me to look at [this action](https://github.com/taiki-e/install-action) which I've seen used elsewhere in our repos and boi was it effective :D Thanks 

<img width="1414" alt="image" src="https://github.com/user-attachments/assets/f2bdb78c-14b5-4e45-bbcc-d7f1c58836ac" />

Reply:
So that `install:wasm-pack:cargo` isn't used anywhere in CI anymore, only for windows developers where binary availability is limited. Is that good for you @adamchalmers?

Reply:
Are we okay with the code scanning warning? It's only used in tests, but it's equivalent to giving outside contributors access, and we decided against that.

Reply:
@jtran good point. Updated to `taiki-e/install-action@v2.48.20` as discussed on Slack